core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreWebViewClient.java

```diff
@@ -121,26 +119,17 @@ public void onPageFinished(WebView view, String url) {
     Log.d(TAG_KIWIX, "invalidUrl = " + invalidUrl);
 
     if (invalidUrl) {
-      onInvalidUrl(view);
+      Log.e(
```

Comment:
Oh the clients are now unified? Great, I'd probably get rid of this log, we already log this url above